docs/guide/security-best-practices.md

```diff
@@ -185,6 +185,30 @@ class SiteController extends Controller
 }
 ```
 
+Disabling CSRF validation in [standalone actions](structure-controllers.md#standalone-actions) must be done in `init()`
+method. Do not place this code into `beforeRun()` method because it won't have effect.
+
+```php
+<?php
+
+namespace app\components;
+
+use yii\base\Action;
+
+class HelloWorldAction extends Action
+{
+    public function init()
+    {
+        parent::init();
+        \Yii::$app->controller->enableCsrfValidation = false;
+    }
+
+    public function run()
+    {
+        return "Hello World";
```

Comment:
@samdark This does not mean we should do the same as in Django. And I found it only after adding, was looking for a better example. I will try to fix it according to your remarks.

> then they're doing it wrong.

Not necessary. Maybe it's just a different philosophy and they think people already understand what CSRF is about when reading this section. Also they have some explanation at the very beginning.